lib/kernel/test/pg_SUITE.erl

```diff
@@ -270,7 +271,32 @@ two(Config) when is_list(Config) ->
     stop_node(TwoPeer, Socket),
     %% hope that 'nodedown' comes before we route our request
     sync(?FUNCTION_NAME),
+
+empty_group_by_remote_leave() ->
+    [{doc, "Empty group should be deleted from nodes."}].
+
+empty_group_by_remote_leave(Config) when is_list(Config) ->
+    {TwoPeer, Socket} = spawn_node(?FUNCTION_NAME, ?FUNCTION_NAME),
+    RemoteNode = rpc:call(TwoPeer, erlang, whereis, [?FUNCTION_NAME]),
+    RemotePid = erlang:spawn(TwoPeer, forever()),
+    % remote join
+    ?assertEqual(ok, rpc:call(TwoPeer, pg, join, [?FUNCTION_NAME, ?FUNCTION_NAME, RemotePid])),
+    sync({?FUNCTION_NAME, TwoPeer}),
+    ?assertEqual([RemotePid], pg:get_members(?FUNCTION_NAME, ?FUNCTION_NAME)),
+    % inspecting internal state is not best practice, but there's no other way to check if the state is correct.
+    {status, _, _, [_, _, _, _, [_, _, {data, [{"State", {state, _, _, #{RemoteNode := {_, RemoteMap}}}}]}]]} 
```

Comment:
There is sys:get_state function that returns state  (easier and shorter to match).

Reply:
Thanks, I force pushed new code with sys:get_state.

I found this issue when I wrote my re-impl of pg in Elixir (https://github.com/jechol/rpg) for local meetup in Seoul, South Korea.